### PR TITLE
Add Bamboo plugins docs 

### PR DESCRIPTION
**Pull request by begonaguereca**
## How's this tested?

Closes https://github.com/github/valet/issues/6202

### CODE REVIEW

File: docs/bamboo/index.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+| [Grails](plugins/Grails.md)                      | actions/setup-java, run, EnricoMi publish-unit-test-result-action |
+| [Grunt](plugins/Grunt.md)                        | actions/setup-node, run                                           |
+| [Gulp](plugins/Gulp.md)                          | actions/setup-node, run                                           |
+| [Inject Variables](plugins/InjectVariables.md)   | run                                                               |
+| [Junit](plugins/Junit.md)                        | EnricoMi/publish-unit-test-result-action                          |
+| [Maven](plugins/Maven.md)                        | actions/setup-java                                                |
+| [Mocha](plugins/Mocha.md)                        | EnricoMi/publish-unit-test-result-action                          |
+| [Ms Build](plugins/MsBuild.md)                   | run                                                               |
+| [MS Test](plugins/MSTest.md)                     | EnricoMi/publish-unit-test-result-action                          |
```
Comment (j-dunham) [inline docs/bamboo/index.md:21]:
```suggestion
| [MS Test](plugins/Mstest.md)                     | EnricoMi/publish-unit-test-result-action                          |
```

File: docs/bamboo/index.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+| [Npm](plugins/Npm.md)                            | actions/setup-node, run                                           |
+| [NUnit Parser](plugins/NunitParser.md)           | EnricoMi/publish-unit-test-result-action                          |
+| [NUnit Runner](plugins/NUnitRunner.md)           | microsoft/vstest-action, EnricoMi/publish-unit-test-result-action |
+| [Repository Branch](plugins/RepositoryBranch.md) | run                                                               |
+| [Repository Commit](plugins/RepositoryCommit.md) | run                                                               |
+| [Repository Push](plugins/RepositoryPush.md)     | run                                                               |
+| [Repository Tag](plugins/RepositoryTag.md)       | run                                                               |
+| [Scp](plugins/Scp.md)                            | appleboy/scp-action                                               |
+| [SSH](plugins/SSH.md)                            | shimataro/ssh-key-action, run                                     |
```
Comment (j-dunham) [inline docs/bamboo/index.md:31]:
```suggestion
| [SSH](plugins/ssh.md)                            | shimataro/ssh-key-action, run                                     |
```

File: docs/bamboo/index.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+| [Stop Job](plugins/StopJob.md)                   | run                                                               |
+| [TestNG](plugins/TestNG.md)                      | scacap/action-surefire-report                                     |
+| [Unlock Keychain](plugins/UnlockKeychain.md)     | run                                                               |
+| [Visual Studio](plugins/Visual Studio.md)        | seanmiddleditch/gha-setup-vsdevenv, run                           |
```
Comment (j-dunham) [inline docs/bamboo/index.md:35]:
```suggestion
| [Visual Studio](plugins/VisualStudio.md)        | seanmiddleditch/gha-setup-vsdevenv, run                           |
```

File: docs/bamboo/index.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+## Trigger Mappings
```
Comment (j-dunham) [inline docs/bamboo/index.md:37]:
```suggestion
# Trigger Mappings
```
To make it match the `Task Mapping` heading

**Review by j-dunham — APPROVED** [review]
🎉

File: docs/bamboo/index.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+| [Ant](plugins/Ant.md)                            | actions/setup-java, run                                           |
+| [Artifact Download](plugins/ArtifactDownload.md)  | actions/download-artifact                                        |
+| [Bower](plugins/Bower.md)                        | actions/setup-node, run                                           |
+| [Checkout](plugins/Checkout.md)                  | actions/checkout                                                  |
+| [Clean](plugins/Clean.md)                        | run                                                               |
+| [Command](plugins/Command.md)                    | run                                                               |
+| [Docker CLI](plugins/Dockercli.md)               | run                                                               |
+| [Fastlane](plugins/Fastlane.md)                  | ruby/setup-ruby, run                                              |
+| [Grails](plugins/Grails.md)                      | actions/setup-java, run, EnricoMi publish-unit-test-result-action |
```
Comment (simonsanchez) [inline docs/bamboo/index.md:13]: Added slash to match the usage of EnricoMi throughout the rest of this file
```suggestion
| [Grails](plugins/Grails.md)                      | actions/setup-java, run, EnricoMi/publish-unit-test-result-action |
```

**Review by simonsanchez — APPROVED** [review]

**Review by ethanis — APPROVED** [review]

File: docs/bamboo/index.md (hunk 2 above)
Comment (begonaguereca) [reply docs/bamboo/index.md:31]: Ok it currently is lowercase but should at least first letter be capitalized to match the other providers?

File: docs/bamboo/index.md (hunk 1 above)
Comment (begonaguereca) [reply docs/bamboo/index.md:21]: Updated the file path

File: docs/bamboo/index.md (hunk 2 above)
Comment (begonaguereca) [reply docs/bamboo/index.md]: I matched it to the others

File: docs/bamboo/plugins/Ant.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+    env:
+      JAVA_OPTS: "-Xmx256m -Xms128m"
+  - name: Test Task
+    run: ant clean test
+    env:
+      JAVA_OPTS: "-Xmx256m -Xms128m"
+    working-directory: test
+```
+
+## Unsupported Options
```
Comment (begonaguereca) [inline docs/bamboo/plugins/Ant.md:42]:
```suggestion
## Unsupported Options

- None 
```

File: docs/bamboo/plugins/Node.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+
+```yaml
+- run: node --foo bar app.js
+  env:
+    HI: BYE
+  working-directory: src
+  if: env.ABC != ''
+```
+
+## Unsupported Options
```
Comment (begonaguereca) [inline docs/bamboo/plugins/Node.md:32]:
```suggestion
## Unsupported Options
- None
```

**Pull request merged**